### PR TITLE
Whole lot of changes, mainly in typos

### PR DESCRIPTION
**Pull request by brickviking**
Again, let me know if you find anything odd that I may have misinterpreted.

Cheers, brickviking

### CODE REVIEW

File: Core/Object Arts/Dolphin/MVP/Views/Scintilla/UI.Scintilla.TextStyle.cls [hunk 1]
```diff
@@ -29,7 +29,7 @@ Note that the `#braceHighlight` and `#mismatchedBrace` styles are normally imple
...
+Up to 256 styles can be defined per view. These have symbolic names for use in Smalltalk code, and have ids allocated automatically. The predefined styles listed above have ids which Scintilla recognises specifically, with ids between `STYLE_DEFAULT` (32) and `STYLE_LASTPREDEFINED` (39). In addition style id 0 is reserved for "normal", which (somewhat confusingly given the existance of `STYLE_DEFAULT`) is the style applied to text by default and from which all the other styles inherit any unspecified properties. See the Scintilla docs for further details of the predefined styles. All other styles are user-defined, can have any symbolic name, and are allocated an unused id in the range 1..255.
```
Comment (blairmcg) [inline Core/Object Arts/Dolphin/MVP/Views/Scintilla/UI.Scintilla.TextStyle.cls:32]: id's is an abbreviation of identifiers, so should have an apostrophe to denote contraction. Or expand it.

File: Core/Object Arts/Dolphin/System/Compiler/Kernel.StProgramNode.cls [hunk 2]
```diff
@@ -241,7 +241,7 @@ isReturn
 	^false!
 
 isSelfVariable
-	"Answer whether the variable described by the receiver is the special variables 'super'."
+	"Answer whether the variable described by the receiver is the special variable 'super'."
```
Comment (blairmcg) [inline Core/Object Arts/Dolphin/System/Compiler/Kernel.StProgramNode.cls:245]: 'self'

File: Core/Object Arts/Dolphin/System/Compiler/Kernel.StSelfVariableNode.cls [hunk 3]
```diff
@@ -16,7 +16,7 @@ acceptVisitor: aProgramNodeVisitor
 	^aProgramNodeVisitor visitSelfVariableNode: self!
 
 isSelfVariable
-	"Answer whether the variable described by the receiver is the special variables 'super'."
+	"Answer whether the variable described by the receiver is the special variable 'super'."
```
Comment (blairmcg) [inline Core/Object Arts/Dolphin/System/Compiler/Kernel.StSelfVariableNode.cls:19]: 'self'

File: Core/Object Arts/Dolphin/System/Compiler/Kernel.StProgramNode.cls (hunk 2 above)
Comment (brickviking) [reply Core/Object Arts/Dolphin/System/Compiler/Kernel.StProgramNode.cls:245]: Whoops... I guess I missed that one.

File: Core/Object Arts/Dolphin/MVP/Views/Control Bars/UI.ControlBarAbstract.cls [hunk 4]
```diff
@@ -63,7 +63,7 @@ arrangementOf: aView put: anObject
 			self invalidateLayout]!
 
 canAcceptSubViews
-	"Answer true if the receiver can accept aView sub views"
+	"Answer true if the receiver can accept aView sub-views"
```
Comment (blairmcg) [inline Core/Object Arts/Dolphin/MVP/Views/Control Bars/UI.ControlBarAbstract.cls:66]: I failed to notice the extraneous `aView` here before, even though I commented about the hyphen

File: Core/Object Arts/Dolphin/MVP/Views/Control Bars/UI.Toolbar.cls [hunk 5]
```diff
@@ -678,7 +678,7 @@ tbEnableButton: anIntegerId state: enableBoolean
 	This is to prevent a walkback appearing in the idle processing of #validUserInterface
 	when enabling/disabling toolbar buttons while a view is being closed. The problem only
 	occurs when a view is closed by a process other than the user interface process 
-	(e.g the Debugger). Unfotunately it is just not possible to test the toolbar with #isOpen 
+	(e.g the Debugger). Unfotunately it is not possible to test the toolbar with #isOpen
```
Comment (blairmcg) [inline Core/Object Arts/Dolphin/MVP/Views/Control Bars/UI.Toolbar.cls:681]: I also failed to notice the `Unfotunately` typo when commenting about removal of 'just'

Comment (brickviking) [reply Core/Object Arts/Dolphin/MVP/Views/Control Bars/UI.Toolbar.cls:681]: Bah. I'm losing my touch, I should have spotted that far sooner.

**Pull request merged**